File: online-city-tickets-frontend/src/app/api/models/time-limited-ticket-offer-dto.ts
/* tslint:disable */
/* eslint-disable */
import { BaseTicketOfferDto } from '../models/base-ticket-offer-dto';
export type TimeLimitedTicketOfferDto = BaseTicketOfferDto & {
'scope': 'time-limited';
'durationMinutes': number;
};

File: online-city-tickets-backend/src/main/java/org/pwr/onlinecityticketsbackend/dto/TimeLimitedTicketOfferDto.java
package org.pwr.onlinecityticketsbackend.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

@Data
@SuperBuilder
@EqualsAndHashCode(callSuper = true)
@NoArgsConstructor
@AllArgsConstructor
public class TimeLimitedTicketOfferDto extends BaseTicketOfferDto {
    private final String scope = "time-limited";
    private int durationMinutes;
}
